### ISSUE TITLE
The xiaomi_viomi integration does not support YAML setup ? 

### ISSUE THREAD

**Issue by Tontze** [nergal/homeassistant-vacuum-viomi#46]
After latest updates to HAOS: 

Logger: homeassistant.setup
Source: setup.py:233
First occurred: 20:06:15 (1 occurrences)
Last logged: 20:06:15

The xiaomi_viomi integration does not support YAML setup, please remove it from your configuration

What should be done ?

**Comment by Tontze** [comment]
Ok well, i deleted everything under the card yaml and it generated excactly same config back and now it works ..

**Comment by Smoetzak** [comment]
> Ok well, i deleted everything under the card yaml and it generated excactly same config back and now it works ..

Do you mean deleted under the custom_components folder, or deleted the vacuum card from the lovelace gui?
Because I have the same issue.

**Comment by SunilKhusal** [comment]
Hi All, any instructions with the above issue as I have the same problem.

**Comment by Smoetzak** [comment]
I've deleted the viomi folder under custom_components and redownloaded it through HACS. It's now working fine again.

**Comment by Tontze** [comment]
> > Ok well, i deleted everything under the card yaml and it generated excactly same config back and now it works ..
> 
> Do you mean deleted under the custom_components folder, or deleted the vacuum card from the lovelace gui? Because I have the same issue.

No, i opened lovelace card and deleted all yaml code Under it.

**Comment by salvadornico** [comment]
I'm getting the same notification even though I've already deleted everything related to the component from configuration.yaml
Searching for `xiaomi_viomi` shows nothing except for files in the custom component itself. Does anyone else have the same problem and know how to stop the notification?

**Comment by Tontze** [comment]
> I'm getting the same notification even though I've already deleted everything related to the component from configuration.yaml Searching for `xiaomi_viomi` shows nothing except for files in the custom component itself. Does anyone else have the same problem and know how to stop the notification?

Yes, i have this error too again. nothing in configuration.yaml, but i have xiaomi-vacuum-map-card where i have 👍 
`
entity: vacuum.viomi
vacuum_platform: marotoweb/viomise
`

But i cannot remove those, as card will surely not work without :o

**Comment by herberthobregon** [comment]
I have the same issue. Does anyone know any solution?

**Comment by Tontze** [comment]
I just ignored it.

**Comment by herberthobregon** [comment]
> I just ignored it. 

It makes me anxious to see that error every time I go into the settings. 🤣🤣

Also I updated the manifest

"python-miio>=0.5.12"

**Comment by 2137home** [comment]
any new solutions?

**Comment by Stefos13** [comment]
Any news on this? I have the same problem and is annoying..
